src/storage/src/hummock/compactor.rs

```diff
@@ -160,6 +160,7 @@ impl Compactor {
             // VNode mappings are not required when compacting shared buffer to L0
             vnode_mappings: vec![],
             compaction_group_id: StaticCompactionGroupId::StateDefault.into(),
+            exist_table_id: vec![],
```

Comment:
This will cause shared buffer compaction producing empty SSTs. We should disable the check for dropped table ids for shared buffer compaction.

Reply:
Ok, i think that we can distinguish shared_buffer_compation and normal_compaction by using strategy like CompactionFilter (next commit)